src/utils.js

```diff
@@ -0,0 +1,13 @@
+export function saveToLocal(name, data) {
+  localStorage.setItem(name, JSON.stringify(data))
+}
+
+export function loadFromLocal(name) {
+  const jsonString = localStorage.getItem(name)
+  const data = JSON.parse(jsonString)
+  return data
+}
+
+export function get(sel) {
```

Comment:
Are you using this get function anywhere yet?

Reply:
No, it's only for this file

Reply:
oh right, I don't need this